XRTK-Core/Packages/com.xrtk.core/Tests/Services/ITestExtensionService1.cs

```diff
@@ -1,9 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.Collections.Generic;
 using XRTK.Interfaces;
 
 namespace XRTK.Tests.Services
 {
-    internal interface ITestExtensionService1 : IMixedRealityExtensionService, ITestService { }
+    internal interface ITestExtensionService1 : IMixedRealityExtensionService
```

Comment:
I think that since this is still an extension service we can keep the previous inheritance from `ITestService`.

We probably want to run the same set of tests against each.

Reply:
Extension Services are going